guest/handle/handle.go

```diff
@@ -34,3 +42,26 @@ func RejectWaitingPod(uid string) bool {
 	runtime.KeepAlive(uid)
 	return wasmBool == 1
 }
+
+func GetWaitingPod(uid string) guestapi.WaitingPod {
+	ptr, size := mem.StringToPtr(uid)
+
+	// Wrap to avoid TinyGo 0.28: cannot use an exported function as value
+	podBytes := mem.SendAndGetPodBytes(ptr, size, func(input_ptr, input_size, ptr uint32, limit mem.BufLimit) {
```

Comment:
any reason why we have to fetch pod in a different way from others, with a new function `SendAndGetPodBytes`?

e.g., when we fetch node, we do like this:
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/b7b0d94b38c573aa707b11970d5137ba51f5f431/guest/internal/imports/host.go#L73-L77

Reply:
Thank you for your information. There's no reason to create a new function. I will update this.